src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -9,7 +9,7 @@
         <env-var name="DB_USER" value="root" />
         <env-var name="DB_PASS" value="123456789" />
         <env-var name="DB_NAME" value="oauth2demo" />
-        <env-var name="USE_CLOUD_SQL" value="true" />
```

Comment:
Why changed this?

Reply:
In memory setting can reduce the time for launching the application. It is better for me to test and modify the UI.